tests/test_compute_funs.py

```diff
@@ -1660,6 +1660,26 @@ def test_surface_equilibrium_geometry():
     """Test that computing stuff from surface gives same result as equilibrium."""
     names = ["DSHAPE", "HELIOTRON", "NCSX"]
     data = ["A", "V", "a", "R0", "R0/a", "a_major/a_minor"]
+    # TODO: expand this to include all angular derivatives once they are implemented
```

Comment:
in the spirit of decreasing test time we probably don't need to test this against 3 different equilibria, can probably pick just one (a stellarator to get the 3d effects) and can crank down the resolution since the geometry doesnt care if its actually in force balance

Reply:
True, I had just copied the prior test that computed these things